sdk/supercontract_v2_transaction.go

```diff
@@ -0,0 +1,501 @@
+		AutomaticExecutionsNumber: automaticExecutionsNumber,
+	}
+
+	return &tx, nil
+}
+
+func (tx *AutomaticExecutionsPaymentTransaction) GetAbstractTransaction() *AbstractTransaction {
+	return &tx.AbstractTransaction
+}
+
+func (tx *AutomaticExecutionsPaymentTransaction) String() string {
+	return fmt.Sprintf(
+		`
+			"AbstractTransaction": %s,
+			"ContractKey": %s,
+			"AutomaticExecutionsNumber": %d,
+		`,
+		tx.AbstractTransaction.String(),
+		tx.ContractKey.String(),
+		tx.AutomaticExecutionsNumber,
+	)
+}
+
+func (tx *AutomaticExecutionsPaymentTransaction) Bytes() ([]byte, error) {
+	builder := flatbuffers.NewBuilder(0)
+
+	v, signatureV, signerV, deadlineV, fV, err := tx.AbstractTransaction.generateVectors(builder)
+	if err != nil {
+		return nil, err
+	}
+
+	contractKeyB, err := hex.DecodeString(tx.ContractKey.PublicKey)
+	if err != nil {
+		return nil, err
+	}
+	
+	contractKeyV := transactions.TransactionBufferCreateByteVector(builder, contractKeyB)
+
+	transactions.AutomaticExecutionsPaymentTransactionBufferStart(builder)
+	transactions.TransactionBufferAddSize(builder, tx.Size())
+	tx.AbstractTransaction.buildVectors(builder, v, signatureV, signerV, deadlineV, fV)
+
+	transactions.AutomaticExecutionsPaymentTransactionBufferAddContractKey(builder, contractKeyV)
+	transactions.AutomaticExecutionsPaymentTransactionBufferAddAutomaticExecutionsNumber(builder, tx.AutomaticExecutionsNumber)
+
+	t := transactions.TransactionBufferEnd(builder)
+	builder.Finish(t)
+
+	return automaticExecutionsPaymentTransactionSchema().serialize(builder.FinishedBytes()), nil
+}
+
+func (tx *AutomaticExecutionsPaymentTransaction) Size() int {
+	return AutomaticExecutionsPaymentHeaderSize
+}
+
+type automaticExecutionsPaymentTransactionDTO struct {
+	Tx struct {
+		abstractTransactionDTO
+		ContractKey 				string `json:"contractKey"`
+		AutomaticExecutionsNumber 	uint32 `json:"automaticExecutionsNumber"`
+	} `json:"transaction"`
+	TDto transactionInfoDTO `json:"meta"`
+}
+
+func (dto *automaticExecutionsPaymentTransactionDTO) toStruct(*Hash) (Transaction, error) {
+	info, err := dto.TDto.toStruct()
+	if err != nil {
+		return nil, err
+	}
+
+	atx, err := dto.Tx.abstractTransactionDTO.toStruct(info)
+	if err != nil {
+		return nil, err
+	}
+
+	contractKey, err := NewAccountFromPublicKey(dto.Tx.ContractKey, atx.NetworkType)
+	if err != nil {
+		return nil, err
+	}
+
+	return &AutomaticExecutionsPaymentTransaction{
+		*atx,
+		contractKey,
+		dto.Tx.AutomaticExecutionsNumber,
+	}, nil
+}
+
+// manual call transaction
+func NewManualCallTransaction(
+	deadline 				*Deadline,
+	contractKey 			*PublicAccount,
+	fileNameSize 			uint16,
+	functionNameSize 		uint16,
+	actualArgumentsSize 	uint16,
+	executionCallPayment 	Amount,
+	downloadCallPayment 	Amount,
+	servicePaymentsCount 	uint8,
+	fileName 				string,
+	functionName 			string,
+	actualArguments 		string,
+	servicePayments 		[]*MosaicId,
+	networkType 			NetworkType,
+) (*ManualCallTransaction, error) {
```

Comment:
see comment about redundant fields.

Also, looks like this file and some other have problems with formatting. Fix it with `go fmt` tool,  pleas.